src/interface.rs

```diff
@@ -31,14 +33,15 @@ where
     RST: OutputPin,
     DELAY: DelayMs<u8>,
 {
-    pub fn new(cs: CS, busy: BUSY, dc: DC, rst: RST) -> Self {
+    pub fn new(cs: CS, busy: BUSY, dc: DC, rst: RST, delay_ms: u8) -> Self {
```

Comment:
2. This could then be either read from a default value/constant for this specific epd or from the optional delay_ms in WaveshareDisplay::new()